### PR TITLE
OUT-106 Add Embed Type

### PR DESCRIPTION
**Pull request by aatbip**
Ref: https://linear.app/copilotplatforms/issue/OUT-106/add-embed-type-to-client-home

- Adds embed type to the editor

### CODE REVIEW

**Comment by aatbip** [comment]
> Please fix the lint issues causing build fail.
> 
> Most likely it is related to tweaking the prettier config in an earlier commit

Linter is turned off on this [branch](https://github.com/copilot-platforms/client-home/blob/edc2aeb467758cb25190d0cc0ef39a57cdc20d66/next.config.js#L22). Some minor changes are made with Linters in #58. After the merge, linter should get restarted.

File: src/components/tiptap/iframe/ext_iframe.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  renderHTML({ HTMLAttributes }) {
+    return ['div', this.options.HTMLAttributes, ['iframe', HTMLAttributes]]
+  },
+
+  addCommands() {
+    return {
+      setIframe:
+        (options: { src: string }) =>
+        ({ tr, dispatch }) => {
```
Comment (saugatmaharjan) [inline src/components/tiptap/iframe/ext_iframe.ts:68]: I see formating error here @aatbip

**Comment by saugatmaharjan** [comment]
@aatbip please address lint and type warnings.

**Comment by aatbip** [comment]
> @aatbip please address lint and type warnings.

@saugatmaharjan This PR will get merged to #58 which has the linters setup.

**Review by saugatmaharjan — APPROVED** [review]

**Pull request merged**